tlair/src/main/scala/at/forsyte/apalache/tla/lir/io/Printer.scala

```diff
@@ -302,3 +302,148 @@ object SimplePrinter extends Printer {
 object FullPrinter extends Printer {
   override def apply(p_ex: TlaEx): String = ""
 }
+
+object UsableAsIdentifierPrinter extends Printer {
+  val leftBracket = "d"
+  val rightBracket = "b"
```

Comment:
Are these just magic constants? Maybe we should use some ASCII constant like `_COLON_` below? Either way, it would be good to have a comment explaining what they represent.

Reply:
Yes - they are a weird way to print brackets, which maybe is not optimal.
I originally chose this for brevity, since identifiers get quite long, and something like _LEFTBRACKET_ is very long, but maybe it's too obscure to have these

Reply:
Might be fine, but we should check that we don't produce name clashes if these constants are alphabetical characters.